developer/guides/troubleshooting/layout-shift.mdx

```diff
@@ -1,14 +1,14 @@
 ---
 title: Layout shift on initial page load
-description: Learn how to identify and diagnose layout shifts on initial page load in your app
+description: Learn how to fix a layout shift on initial page load in your app
 ---
 
-## What is a layout shift?
+{/* @todo(sasha): Update with a correct link to the blog post once it is ready */}
+{/* <Note>
+For a deeper dive into diagnosing layout shifts, check out our blog post: [Diagnosing a Layout Shift](/blog/diagnosing-layout-shifts)
+</Note> */}
 
-Layout shift is a common issue in web development where elements on a page move unexpectedly,
-causing a poor user experience.
-
-Here is an example of a layout shift:
+Watch the example below to recognize a layout shift caused by a component update during hydration. If your app behaves similarly, follow the steps in this guide.
 
 <Frame>
   <video
```

Comment:
Did the video referenced here get deleted on purpose?

Reply:
Nope :D